.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wanted to ask at the KT meeting about the 2 .idea files and time was up. I will add it to the next meeting!

Reply:
I don't know about those. I based my PR out of David's PR